metadata-models/src/main/pegasus/com/linkedin/ml/metadata/EvaluationData.pdl

```diff
@@ -0,0 +1,24 @@
+namespace com.linkedin.ml.metadata
+
+import com.linkedin.common.DatasetUrn
+
+/**
+ * All referenced datasets would ideally point to any set of documents that provide visibility into the source and composition of the dataset.
+ */
+record EvaluationData {
+
+  /**
+   * What datasets were used to evaluate the model?
+   */
+  datasets: optional array[DatasetUrn]
+
+  /**
+   * Why were these datasets chosen?
+   */
+  motivation: optional string
+
+  /**
+   * How was the data preprocessed for evaluation (e.g., tokenization of sentences, cropping of images, any filtering such as dropping images without faces)?
+   */
+  preProcessing: optional string
```

Comment:
What if multiple preprocessing steps are involved (in order)? Will an array be more suitable?